src/harberger-lib/components/Buy.res

```diff
@@ -290,15 +290,18 @@ module Buy = {
               />
             </>
           : <Rimble.Box>
-              <p className=Styles.textOnlyModalText>
-                {React.string(
-                  "You do not have enough " ++ (currency ++ (" to buy " ++ (tokenIdName ++ "."))),
-                )}
-              </p>
               <p>
                 {("Your current balance is: " ++ (paymentTokenBalance ++ (" " ++ currency)))
                   ->React.string}
               </p>
+              <p className=Styles.textOnlyModalText>
+                {`You do not have enough ${currency} to buy ${tokenIdName}.`->React.string}
+              </p>
+              <p className=Styles.textOnlyModalText>
+                {`You can either move your mainnet Ethereum dai to matic network using `->React.string}
+                <a href="https://wallet.matic.network/"> {"matic wallet"->React.string} </a>
+                {` or use your credit card or a bank transfer using the button below:`->React.string}
+              </p>
```

Comment:
Before: 
![image](https://user-images.githubusercontent.com/6032276/104943289-369f4d00-59be-11eb-86fc-b0bc4a3d081d.png)

After:
![image](https://user-images.githubusercontent.com/6032276/104943507-8a119b00-59be-11eb-90e3-e764716e6547.png)


Kind of looks fonkong, but I just want this to get released before our next batch of wildcards...